packages/explat-react-client/src/index.ts

```diff
@@ -1,2 +1,50 @@
-// Stub index
-export default null;
+import { useState, useEffect } from 'react'
+
+import type { ExPlatClient, ExperimentAssignment } from '@automattic/explat-client'
+
+interface ExPlatReactClient {
+    /**
+     * An ExPlat useExperiment hook.
+     *
+     * NOTE: Doesn't obey ExperimentAssignment TTL in order to keep stable UX.
+     *
+     * @returns [isExperimentAssignmentLoading, ExperimentAssignment | null]
+     */
+    useExperiment: (experimentName: string) => [boolean, ExperimentAssignment | null]
+}
+
+export default function createExPlatReactClient(exPlatClient: ExPlatClient): ExPlatReactClient {
```

Comment:
I'm not sure about the names here. It's pretty different from the "real" client that we're wrapping. 

Maybe we should call the interface `ExPlatReactHooks` and this function `createExPlatReactHooks`? We may want to rename the package as well.

Reply:
It is going to have the `<Experiment>` component as well so maybe: `ExPlatReact`.

Reply:
Maybe `ExPlatReactHelpers`?

Reply:
`ExPlatClientReactHelpers`?

Reply:
Sounds good :slightly_smiling_face: